### PR TITLE
Hacking Instructor Local XSS Fix

### PR DESCRIPTION
**Pull request by J12934**
Updated expected text to match recommended text.

The step will only be resolved when the *exact* text is input.

### CODE REVIEW

**Comment by bkimminich** [comment]
The recommendation shouldn't have the closing tag actually. So the expected text is the correct one.

**Comment by J12934** [comment]
Mh ok, interesting.
Why is that? Seems hard to follow the instructions when copy and pasting as this won’t trigger the next hint.

**Comment by bkimminich** [comment]
The hint text in line 85 is just plain wrong. Doesn't match what the Score Board tells you to use as payload.

**Comment by J12934** [comment]
Ahh ok.
I've updated the text in the instruction.
